app/services/trainees/create_from_hesa.rb

```diff
@@ -232,7 +205,7 @@ def itt_end_date
     # HESA do not distinguish between the ITT start date and the trainee
     # start date, so we're setting both to the ITT start date.
     def trainee_start_date
-      itt_start_date
+      hesa_trainee[:trainee_start_date] || itt_start_date
```

Comment:
I think the above comment can be removed as well, as it no longer just `itt_start_date`